packaging/docker/start-node.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+if [[ -z "$HTTP_RPC" ]]; then
```

Comment:
@Mirko-von-Leipzig @CHr15F0x I'm wondering if it might be good to add support for these env vars directly into the `pathfinder` code itself? That way this can all be supported

Reply:
That would be ideal. That being said, I have a habit of using starter scripts cause they add a lot of flexibility to minor UX improvements.

Reply:
Where are these env vars coming from? Sorry, my docker-fu is practically zero.

Reply:
They're more or less arbitrary - but I'm talking about the rust code supporting e.g. `ETHEREUM_PASSWORD` in addition to the CLI flag

Reply:
Its definitely possible; I thought having a configuration file and cmd line args would be enough for now. But if its required we can definitely add them in quite easily.

Should likely be `PATHFINDER_XXX` prefixed though?

Reply:
For better or worse env vars are the de-facto method of configuration for deployments 📦 🚢 

@odyslam mitigated this by building them into this startup script but it would likely be better if pathfinder supported these natively